### DISCUSSION TITLE
Unknown media type: Series

### DISCUSSION THREAD

**Discussion by Johnymalina** [IgnisDa/ryot#862] [Q&A] [answered]
Why iam getting this error importing from Jellyfin?

**Comment by IgnisDa** [comment]
I haven't added show support to the jellyfin importer. I plan to work on it soon. If you want, you can join the Discord to track progress.

**Reply by IgnisDa** [reply, marked as answer]
Show support has been added and is working now!

**Reply by Johnymalina** [reply]
Nice. I can confirm that its working. I had to try it two times before i realize that its using Password was it API key earlier or am i just confused? One additional question. Does it mean that also newly added series will be automatically imported as well or is manual import required?

**Reply by IgnisDa** [reply]
Yes earlier it was api key but I switched to password.

For newly added media you're currently watching, you'll have to setup an integration. Import is a one time thing.
